docusaurus.config.js

```diff
@@ -19,6 +19,9 @@ module.exports = {
         hideable: true,
       },
     },
+    markprompt: {
+      projectKey: 'jbhF5LligltdKaJucMjDcWcRodaVpzqE',
```

Comment:
@s-santillan intentional?

Reply:
Yes. It's not the bearer token or the dev token (both of which should be either secrets or not committed at all). I apologize for the confusion.

Reply:
Thank you!